### PR TITLE
reordering open positions sort dropdown

### PR DESCRIPTION
**Pull request by SidelineCory24**
To-do:
**New Order Open Positions**
Featured Positions
Posted Date: Most Recent
Posted Date: Oldest
Bureau: A-Z
Bureau: Z-A
Grade: Low to High
Grade: High to Low
Location: A-Z
Location: Z-A
Position Number: Low to High
Position Number: High to Low
Position Title: A-Z
Position Title: Z-A
TED: Most Recent
TED: Oldest

**New Order PV**
Bureau: A-Z
Bureau: Z-A
Grade: Low to High
Grade: High to Low
Location: A-Z
Location: Z-A
Position Number: Low to High
Position Number: High to Low
Position Title: A-Z
Position Title: Z-A
TED: Most Recent
TED: Oldest

### CODE REVIEW

File: src/Constants/Sort.js [hunk 1]
```diff
@@ -29,9 +33,6 @@ export const POSITION_SEARCH_SORTS_DYNAMIC = {
   options: [
     ...POSITION_SEARCH_SORTS.options.map((m) => {
       const obj = { ...m };
-      if (obj.value === '-position__bureau') {
-        obj.value = 'position__bureau';
-      }
```
Comment (SidelineCory24) [inline src/Constants/Sort.js:34]: Removing to get `Bureau Z-A` sort to work. I'm not sure why this was added originally.

Comment (SidelineCory24) [reply src/Constants/Sort.js:34]: tested and verified in dev1 that both `Bureau A-Z` and `Bureau Z-A` sorts work successfully

**Comment by SidelineCory24** [comment]
> Were all new sorts verified in dev1?

Yes. I verified this on Friday that they worked in swagger. I can double check again if needed.

**Comment by elizabeth-jimenez** [comment]
> > Were all new sorts verified in dev1?
> 
> Yes. I verified this on Friday that they worked in swagger. I can double check again if needed.

No need if you already checked!

**Review by elizabeth-jimenez — APPROVED** [review]

File: src/Constants/Sort.js [hunk 2]
```diff
@@ -29,9 +33,6 @@ export const POSITION_SEARCH_SORTS_DYNAMIC = {
   options: [
     ...POSITION_SEARCH_SORTS.options.map((m) => {
       const obj = { ...m };
-      if (obj.value === '-position__bureau') {
```
Comment (elizabeth-jimenez) [inline src/Constants/Sort.js:32]: 🙌

**Pull request merged**